app/src/main/res/layout/error_panel.xml

```diff
@@ -72,4 +72,19 @@
         android:visibility="gone"
         tools:visibility="visible" />
 
+    <Button
+        android:id="@+id/error_openinbrowser"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:layout_marginBottom="8dp"
```

Comment:
As you can notice on the screenshot you provided, the margin between the `Open in browser` and `Retry` buttons is not the same as the one between the `Report` and `Retry` buttons. Could you change this?